dimod/include/dimod/adjvectorbqm.h

```diff
@@ -96,159 +98,156 @@ class AdjVectorBQM {
+                }
             }
-          }
-        }
 
-        free(temp_buffer);
+            free(temp_buffer);
 
-        // Restore the diagonal of the original dense matrix
-        #pragma omp for schedule(static)
-        for (size_type v = 0; v < num_variables; ++v) {
-          dense[v * (num_variables + 1)] = dense_diagonal[v];
+// Restore the diagonal of the original dense matrix
+#pragma omp for schedule(static)
+            for (size_type v = 0; v < num_variables; ++v) {
+                dense[v * (num_variables + 1)] = dense_diagonal[v];
+            }
         }
-      }
     }
 
     /// Add one (disconnected) variable to the BQM and return its index.
     variable_type add_variable() {
-        adj.resize(adj.size()+1);
-        return adj.size()-1;
+        adj.resize(adj.size() + 1);
+        return adj.size() - 1;
     }
 
     /// Get the degree of variable `v`.
-    size_type degree(variable_type v) const {
-        return adj[v].first.size();
-    }
+    size_type degree(variable_type v) const { return adj[v].first.size(); }
 
-    [[deprecated("Use AdjVectorBQM::linear(v)")]]
-    bias_type get_linear(variable_type v) const {
-        return linear(v);
-    }
+    [[deprecated("Use AdjVectorBQM::linear(v)")]] bias_type get_linear(
+        variable_type v) const { return linear(v); }
 
-    std::pair<bias_type, bool>
-    get_quadratic(variable_type u, variable_type v) const {
+    std::pair<bias_type, bool> get_quadratic(variable_type u,
+                                             variable_type v) const {
         assert(u >= 0 && u < adj.size());
         assert(v >= 0 && v < adj.size());
         assert(u != v);
 
         auto span = neighborhood(u);
-        auto low = std::lower_bound(span.first, span.second, v,
-                                    utils::comp_v<V, B>);
+        auto low =
+            std::lower_bound(span.first, span.second, v, utils::comp_v<V, B>);
 
         if (low == span.second || low->first != v)
             return std::make_pair(0, false);
         return std::make_pair(low->second, true);
     }
 
-
-    bias_type& linear(variable_type v) {
+    bias_type &linear(variable_type v) {
         assert(v >= 0 && v < adj.size());
         return adj[v].second;
     }
 
-    const bias_type& linear(variable_type v) const {
+    const bias_type &linear(variable_type v) const {
         assert(v >= 0 && v < adj.size());
         return adj[v].second;
     }
 
-    std::pair<outvars_iterator, outvars_iterator>
-    neighborhood(variable_type u) {
+    std::pair<outvars_iterator, outvars_iterator> neighborhood(
+        variable_type u) {
         assert(u >= 0 && u < adj.size());
         return std::make_pair(adj[u].first.begin(), adj[u].first.end());
     }
 
-    std::pair<const_outvars_iterator, const_outvars_iterator>
-    neighborhood(variable_type u) const {
+    std::pair<const_outvars_iterator, const_outvars_iterator> neighborhood(
+        variable_type u) const {
```

Comment:
This feels like it should be indented 8 spaces. Not sure is that due to `ContinuationIndentWidth` or `ConstructorInitializerIndentWidth` not scaled with `IndentWidth` in dwave-ocean-sdk#90.